web/styles/message_row.css

```diff
@@ -667,6 +667,27 @@
     .message-edit-textbox {
         position: relative;
     }
+
+    &:has(.message_edit_content.over_limit),
+    &:has(.message_edit_content.over_limit:focus) {
+        box-shadow: 0 0 0 1pt
+            var(--color-message-content-container-border-over-limit);
+    }
+
+    &:has(.message_edit_content.over_limit.flash) {
+        animation: message-limit-flash 0.5s ease-in-out 3;
+    }
+}
```

Comment:
I think it would be nice to share these styles with the messagebox, something like this:

```css

   #message-content-container:has(.message_edit_content.over_limit),
   #message-content-container:has(.message_edit_content.over_limit:focus),
   .edit-content-container:has(.message_edit_content.over_limit),
   .edit-content-container:has(.message_edit_content.over_limit:focus)  {
        box-shadow: 0 0 0 1pt
            var(--color-message-content-container-border-over-limit);
    }
```

Reply:
I guess you were suggesting something like:

```
#message-content-container:has(.new_message_textarea.over_limit),
#message-content-container:has(.new_message_textarea.over_limit:focus),
```
instead of 

```
#message-content-container:has(.message_edit_content.over_limit),
#message-content-container:has(.message_edit_content.over_limit:focus),
```
I have made the above change. However, I have placed this code to `compose.css` instead of `message_row.css`. 
If you want me to place it back in `message_row.css`, let me know, and I will update the PR.

Reply:
Yeah I think putting them all there is fine, looks great thanks!